packages/block-library/src/list/edit.js

```diff
@@ -14,39 +26,104 @@ export default function ListEdit( {
 	className,
 } ) {
 	const { ordered, values } = attributes;
+	const [ startValue, setStartValue ] = useState( '' );
```

Comment:
I don't understand why there is local state here. Shouldn't it be derived from the `attributes`? And use `setAttributes` to set it?

Reply:
By using the `attributes`, the `start` variable cannot be saved. But `type` and `reversed` can be saved.

Reply:
Why can `start` not be saved? What's difference with the other two?

Reply:
I still do not find the reason...